libs/ngx-nuts-and-bolts/package.json

```diff
@@ -29,5 +29,12 @@
 	},
 	"dependencies": {
 		"tslib": "^2.3.0"
+	},
+	"schematics": "./schematics/src/collection.json",
+	"ng-add": {
+		"save": "dependencies"
+	},
+	"devDependencies": {
+		"typescript": "file:../../node_modules/typescript"
```

Comment:
Why is this necessary?

Reply:
Remnant from previous approach. Deleted.

Reply:
It's... not?